imports/ui/pages/Profile.tsx

```diff
@@ -83,42 +83,84 @@ export const Profile = () => {
+              {profilePicture ? (
+                <img
+                  src={profilePicture}
+                  alt="Profile"
+                  className="w-32 h-32 rounded-full bg-gray-200 border"
+                />
+              ) : (
+                <div className="w-16 h-16 rounded-full bg-gray-200 flex items-center justify-center text-gray-400">
+                  H
+                </div>
+              )}
+              <Button variant="outline">
+                <label htmlFor="profileImage" className="cursor-pointer">
+                  Select a new photo
+                </label>
+              </Button>
+              <input
+                id="profileImage"
+                type="file"
+                accept="image/*"
+                onChange={handleFileChange}
+                className="hidden"
+              />
+            </div>
+            <FormField
+              control={form.control}
+              name="name"
+              render={({ field }) => (
+                <FormItem>
+                  <FormLabel>Name</FormLabel>
+                  <FormControl>
+                    <Input id="name" type="text" {...field} {...register("name")} />
+                  </FormControl>
+                  <FormMessage />
+                </FormItem>
+              )}
+            />
+            <FormField
+              control={form.control}
+              name="email"
+              render={({ field }) => (
+                <FormItem>
+                  <FormLabel>Email</FormLabel>
+                  <FormControl>
+                    <Input id="email" type="email" {...field} {...register("email")} />
+                  </FormControl>
+                  <FormMessage />
+                </FormItem>
+              )}
             />
-          </div>
-        )}
-        <input type="file" accept="image/*" onChange={handleFileChange} />
-        {errors.image && <span>{errors.image.message}</span>}
-        <h2>Your info:</h2>
-        <label>Name:</label>
-        <input type="text" {...register("name")} />
-        {errors.name && <span>{errors.name.message}</span>}
-        <br />
-        <label>Email:</label>
-        <input type="email" {...register("email")} />
-        {errors.email && <span>{errors.email.message}</span>}
-        <br />
-        <button style={{ marginTop: "15px" }} type="submit" disabled={isSubmitting}>
-          {isSubmitting ? "Saving..." : "Save changes"}
-        </button>
-      </form>
-      <PasswordUpdate />
-      <button style={{ marginTop: "20px" }} onClick={handleLogout}>
-        Logout
-      </button>
-      <br />
-      <button onClick={handleDeleteAccount} style={{ marginTop: "20px", color: "red" }}>
-        Delete Account
-      </button>
+            <Button type="submit" className="w-24">
+              {isSubmitting ? "Saving..." : "Save"}
+            </Button>
+          </form>
+        </Form>
+      </div>
+      {user?.services?.password && <PasswordUpdate />}
```

Comment:
dont expose the password in the front end, this is insecure, create a logic in the backend, for instance: withPassword/hasPassword: boolean, then use it here